tests/publiccloud/prepare_tools.pm

```diff
@@ -36,7 +36,7 @@ sub run {
     for my $repo (split(/\s+/, $tools_repo)) {
         zypper_call('ar ' . $repo);
     }
-    zypper_call('--gpg-auto-import-keys -q in python3-ipa python3-ipa-tests git-core ntp');
+    zypper_call('--gpg-auto-import-keys -q in python3-img-proof python3-img-proof-tests git-core ntp');
```

Comment:
Lets also change the `$default_tools_repos` to  `https://build.suse.de/project/show/Devel:PubCloud:CI`.

Reply:
oh It is already there :)